pom.xml

```diff
@@ -77,6 +77,7 @@
         <version.log4j>1.2.16</version.log4j>
         <version.log4j-extras>1.0</version.log4j-extras>
         <version.lucene>7.5.0</version.lucene>
+        <version.microservice.accumulo-utils>1.0-SNAPSHOT</version.microservice.accumulo-utils>
```

Comment:
We need to figure out what to do for things like this. The services folder isn't built by default (though you can do it by adding -Dservices) so a person just building datawave wouldn't have this snapshot dependency. The idea is that the artifacts would be pulled from maven central (or for now our hackish git branch), much like they would for any third party library. However, that would necessitate releasing the dependent artifact. Maybe an interim solution is add module dependencies just for the snapshot artifacts? (Or release a "beta" version of the artifact is another option.) Thoughts?

Reply:
Yeah, I wasn't sure how we wanted to do this. Adding module dep temporarily seems ok I guess, as long as it goes away before a DW release is cut. But maybe beta releases are the cleaner option, even if a little more of a hassle, since that'll maintain the boundary between services/* artifacts and everything else?

Reply:
Well, remember the release of the underlying dependency can be completely independent of the datawave release. The problem with having a snapshot version is that there's no place for someone who does a git clone on the repo and builds it to get the snapshot (without the special knowledge to build the services modules). I'm thinking beta release. Maybe once the PR is approved, merge it in and immediately make the beta release of the library and update the dependency in the top level pom?

Reply:
That should work

Reply:
Ok, given that I have a PR that will be merged into this same module, and am planning additional work that will go there. Maybe the best solution for now is to add `<module>services/accumulo-utils</module>` to the modules section of the top level pom. Then when we're ready to make the first release, we'll tag the accumulo-utils module, update the version here to 1.0, and remove the module specification. This way, someone downloading and building datawave while it's still a snapshot version won't have problems building.  If you don't mind making/testing that change, then we can merge this in.

Reply:
Done